emukit/core/bandit_parameter.py

```diff
@@ -0,0 +1,140 @@
+# Copyright 2020 Opsani, Inc. or its affiliates. All Rights Reserved.
```

Comment:
I honestly have no idea what the rules are around copyright notices. We'll need someone from Amazon to check on this one (I am no longer working there to check myself).

Reply:
Looking into it.

Reply:
Checked, this is fine. Thanks for flagging Andrei!

Reply:
I am pleasantly surprised. Thanks for checking @MashaNaslidnyk